src/libcephfs.cc

```diff
@@ -331,6 +338,7 @@ struct ceph_mount_info
   bool mounted;
   bool inited;
   StandaloneClient *client;
```

Comment:
Your test code should not use libcephfs.h at all.

Reply:
Okay, but then how do I do it then? Is there any other way?

Reply:
Is there any existing code that I can look at?

Reply:
You can follow `test/client/TestClient.h` and `test/client/alternate_name.cc`.

Reply:
Thanks @lxbsz !

Reply:
Thanks @batrick and @lxbsz, created a new header file named `TestNewOpsClient.h` under `client/test`. PTAL.